2024-03-19 06:45:19,115 INFO    MainThread:23664 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-03-19 06:45:19,115 INFO    MainThread:23664 [wandb_setup.py:_flush():76] Configure stats pid to 23664
2024-03-19 06:45:19,115 INFO    MainThread:23664 [wandb_setup.py:_flush():76] Loading settings from C:\Users\CYP\.config\wandb\settings
2024-03-19 06:45:19,115 INFO    MainThread:23664 [wandb_setup.py:_flush():76] Loading settings from c:\Users\CYP\Documents\GitHub\AIAKI\wandb\settings
2024-03-19 06:45:19,115 INFO    MainThread:23664 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-19 06:45:19,115 INFO    MainThread:23664 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-19 06:45:19,116 INFO    MainThread:23664 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-19 06:45:19,116 INFO    MainThread:23664 [wandb_init.py:_log_setup():526] Logging user logs to c:\Users\CYP\Documents\GitHub\AIAKI\wandb\run-20240319_064519-wygk3k82\logs\debug.log
2024-03-19 06:45:19,116 INFO    MainThread:23664 [wandb_init.py:_log_setup():527] Logging internal logs to c:\Users\CYP\Documents\GitHub\AIAKI\wandb\run-20240319_064519-wygk3k82\logs\debug-internal.log
2024-03-19 06:45:19,116 INFO    MainThread:23664 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000002034B76F4C0>
2024-03-19 06:45:19,116 INFO    MainThread:23664 [wandb_init.py:init():566] calling init triggers
2024-03-19 06:45:19,116 INFO    MainThread:23664 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'batch_size': 32, 'n_units': 24, 'learning_rate': 0.001, 'lr_decay': 0.9542684862960304, 'lr_step': 1, 'ns': 2, 'loss': 'smooth_l1', 'update_freq': 8, 'PPO': 0, 'CQL': 0}
2024-03-19 06:45:19,116 INFO    MainThread:23664 [wandb_init.py:init():591] re-initializing run, found existing run on stack: dx5n58gz
2024-03-19 06:45:19,117 INFO    MainThread:23664 [wandb_run.py:_finish():1970] finishing run dahs/IMV_LSTM_Rescue_new/dx5n58gz
2024-03-19 06:45:19,117 INFO    MainThread:23664 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-03-19 06:45:19,117 INFO    MainThread:23664 [wandb_run.py:_restore():2193] restore
2024-03-19 06:45:19,117 INFO    MainThread:23664 [wandb_run.py:_restore():2199] restore done
2024-03-19 06:45:22,455 INFO    MainThread:23664 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-03-19 06:45:22,458 INFO    MainThread:23664 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-03-19 06:45:22,470 INFO    MainThread:23664 [wandb_run.py:_footer_sync_info():3825] logging synced files
2024-03-19 06:45:22,585 INFO    MainThread:23664 [wandb_init.py:init():616] starting backend
2024-03-19 06:45:22,585 INFO    MainThread:23664 [wandb_init.py:init():620] setting up manager
2024-03-19 06:45:22,587 INFO    MainThread:23664 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-03-19 06:45:22,588 INFO    MainThread:23664 [wandb_init.py:init():628] backend started and connected
2024-03-19 06:45:22,591 INFO    MainThread:23664 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-03-19 06:45:22,591 INFO    MainThread:23664 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-03-19 06:45:22,591 INFO    MainThread:23664 [wandb_init.py:init():720] updated telemetry
2024-03-19 06:45:22,613 INFO    MainThread:23664 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-03-19 06:45:24,000 INFO    MainThread:23664 [wandb_run.py:_on_init():2262] communicating current version
2024-03-19 06:45:24,035 INFO    MainThread:23664 [wandb_run.py:_on_init():2271] got version response upgrade_message: "wandb version 0.16.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-03-19 06:45:24,036 INFO    MainThread:23664 [wandb_init.py:init():804] starting run threads in backend
2024-03-19 06:45:24,150 INFO    MainThread:23664 [wandb_run.py:_console_start():2241] atexit reg
2024-03-19 06:45:24,150 INFO    MainThread:23664 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-03-19 06:45:24,150 INFO    MainThread:23664 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-03-19 06:45:24,150 INFO    MainThread:23664 [wandb_run.py:_redirect():2186] Redirects installed.
2024-03-19 06:45:24,151 INFO    MainThread:23664 [wandb_init.py:init():847] run started, returning control to user process
2024-03-19 08:08:51,951 INFO    MainThread:23664 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-03-19 08:08:51,952 INFO    MainThread:23664 [wandb_setup.py:_flush():76] Configure stats pid to 23664
2024-03-19 08:08:51,952 INFO    MainThread:23664 [wandb_setup.py:_flush():76] Loading settings from C:\Users\CYP\.config\wandb\settings
2024-03-19 08:08:51,952 INFO    MainThread:23664 [wandb_setup.py:_flush():76] Loading settings from c:\Users\CYP\Documents\GitHub\AIAKI\wandb\settings
2024-03-19 08:08:51,952 INFO    MainThread:23664 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-19 08:08:51,952 INFO    MainThread:23664 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-19 08:08:51,954 INFO    MainThread:23664 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-19 08:08:51,954 INFO    MainThread:23664 [wandb_init.py:_log_setup():526] Logging user logs to c:\Users\CYP\Documents\GitHub\AIAKI\wandb\run-20240319_080851-30yj0v6x\logs\debug.log
2024-03-19 08:08:51,954 INFO    MainThread:23664 [wandb_init.py:_log_setup():527] Logging internal logs to c:\Users\CYP\Documents\GitHub\AIAKI\wandb\run-20240319_080851-30yj0v6x\logs\debug-internal.log
2024-03-19 08:08:51,954 INFO    MainThread:23664 [wandb_init.py:init():566] calling init triggers
2024-03-19 08:08:51,954 INFO    MainThread:23664 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'batch_size': 256, 'n_units': 37, 'learning_rate': 0.0005, 'lr_decay': 0.8897563156083651, 'lr_step': 1, 'ns': 10, 'loss': 'mse', 'update_freq': 45, 'PPO': 0, 'CQL': 0}
2024-03-19 08:08:51,954 INFO    MainThread:23664 [wandb_init.py:init():591] re-initializing run, found existing run on stack: wygk3k82
2024-03-19 08:08:51,955 INFO    MainThread:23664 [wandb_run.py:_finish():1970] finishing run dahs/IMV_LSTM_Rescue_new/wygk3k82
2024-03-19 08:08:51,955 ERROR   MainThread:23664 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-03-19 08:08:51,955 INFO    MainThread:23664 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-03-19 08:08:51,955 INFO    MainThread:23664 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-03-19 08:08:51,956 INFO    MainThread:23664 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-03-19 08:08:51,956 INFO    MainThread:23664 [wandb_run.py:_restore():2193] restore
2024-03-19 08:08:51,956 INFO    MainThread:23664 [wandb_run.py:_restore():2199] restore done
2024-03-19 08:08:56,912 INFO    MainThread:23664 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-03-19 08:08:56,916 INFO    MainThread:23664 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-03-19 08:08:56,920 INFO    MainThread:23664 [wandb_run.py:_footer_sync_info():3825] logging synced files
